includes/payment-method/class-card.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Class Sepa
+ *
+ * @package WCPay\Payment_Gateway
+ */
+
+namespace WCPay\Payment_Method;
```

Comment:
Using `Gateways` instead of `Payment_Method` would better express intention here.

Reply:
There are already other opinions on the name in this conversation p1617864347114400/1617748545.084400-slack-CGGCLBN58 . Maybe we should put it to a vote? I really don't have an opinion.

Reply:
Sounds good, I'd suggest using wcpay channel for the voting. We can pack the changes based on voting into a follow-up PR as well.